src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -34,6 +39,64 @@ public void setup() {
     }
 
     @Test
+    public void testDownloadTwoGoogleKeywordsBelongsToSameCampaignId(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes"),
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Paused").withText("shirts")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+
+        assertThat(kenshooKeywords, hasSize(2));
+    }
+
+    @Test
+    public void testDownloadActiveGoogleKeyword(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+        assertThat(kenshooKeywords.get(0), isKeyword(withIdInTarget(1), withStatus("Active"), withText("shoes")));
```

Comment:
only assert for the Active status.
don't mind the 'shoes' and idInTarget in this scenario.
You will have other scenarios for those.